README.md

```diff
@@ -68,6 +68,6 @@ We're hiring developers, support people, and product managers all the time. Plea
 * [Youtube](https://www.youtube.com/channel/UCin9nv7mUjoqrRiwrzS5UVQ)
 * [Email Newsletter](https://rocket.chat/newsletter)
 
-Any other questions, reach out to us at [contact@rocket.chat](contact@rocket.chat). We’d happy to help!
+Any other questions, reach out to us at [contact@rocket.chat](https://rocket.chat/contact). We’d happy to help!
```

Comment:
```suggestion
Any other questions, reach out to us at [our website](https://rocket.chat/contact) or you can email us directly at [contact@rocket.chat](mailto:contact@rocket.chat). We’d happy to help!
```

Reply:
I made the requested changes :+1: 
Please review!